### PR TITLE
feat(article-table): Add article table

### PR DESCRIPTION
**Pull request by matijs**
- [x] add ArticleTable molecule
- [x] clean up table atom
- [x] clean up TableCompare organism
- [x] relocate table atom SCSS
- [x] update snapshots & build artefacts
- [x] release new DS version

### CODE REVIEW

File: src/js/components/molecules/article-table/data/article-table.data.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </tr>
+        <tr>
+            <td>row 3:1</td><td>row 3:2</td> <td>row 3:3</td><td>row 3:4</td>
+        </tr>
+        <tr>
+            <td>row 4:1</td><td>row 4:2</td> <td>row 4:3</td><td>row 4:4</td>
+        </tr>
+    </tbody>
+</table>
+        `.trim(),
```
Comment (zwaardje) [inline src/js/components/molecules/article-table/data/article-table.data.ts]: Why trim your own prefilled data?

Comment (matijs) [reply src/js/components/molecules/article-table/data/article-table.data.ts]: habit…

Comment (matijs) [reply src/js/components/molecules/article-table/data/article-table.data.ts]: fixed

File: src/js/components/molecules/article-table/data/article-table.data.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/** @format */
```
Comment (dan-entous) [inline src/js/components/molecules/article-table/data/article-table.data.ts]: i prefer splitting up the examples and placing them an example folder under data and then pull them in; this illustrates what i mean: `src/js/components/organisms/overview/data/overview.data.ts`

File: src/js/components/molecules/article-table/article-table.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { ArticleTableProps } from './article-table.models';
+import WithRender from './article-table.vue.html';
+
+@WithRender
+@Component({
+    name: 'ArticleTable',
+})
+export class ArticleTable extends Vue implements ArticleTableProps {
+    @Prop({ type: String, required: true })
```
Comment (dan-entous) [inline src/js/components/molecules/article-table/article-table.ts]: 🐜 prefer to see the prop statement on one line if possible

File: src/js/components/molecules/article-table/article-table.vue.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<!-- @format -->
+
+<div class="article-table" v-html="table" />
```
Comment (dan-entous) [inline src/js/components/molecules/article-table/article-table.vue.html]: locally i changed this to `<table` and it worked as well without adding a table in a table.

File: src/js/components/molecules/article-table/article-table.ts (hunk 3 above)
Comment (matijs) [reply src/js/components/molecules/article-table/article-table.ts]: I think prettier put it on 2…

File: src/js/components/molecules/article-table/article-table.vue.html (hunk 4 above)
Comment (matijs) [reply src/js/components/molecules/article-table/article-table.vue.html]: I'll give it a go…

File: src/js/components/molecules/article-table/article-table.ts (hunk 3 above)
Comment (dan-entous) [reply src/js/components/molecules/article-table/article-table.ts]: it should use 120 characters per line, but okay.

**Pull request merged**